.ruby-version

```diff
@@ -1 +1 @@
-2.6.3
+2.7.3
```

Comment:
With rvm and setup-ruby if you specify 2.7 here it will mean the latest 2.7 version.  You can achieve the same thing with rbenv by creating a symlink from 2.7 -> 2.7.3 but you have to update it manually when there's a newer version.

Reply:
Seems to work fine with chruby. I'll change.